test/evil-surround-test.el

```diff
@@ -55,10 +55,21 @@
       :visual-end nil
       "argument1 argument2"
       (turn-on-evil-surround-mode)
+      (c-mode)
```

Comment:
Why did you activate `c-mode` in this test here?

Reply:
I wasn't shure what the default mode is, and to make shure that it's non of the lispy modes, i chose c-mode very basic and present everywhere. Feel free to chane it though.

Reply:
That's ok.